dockers/docker-orchagent/supervisord.conf

```diff
@@ -3,6 +3,12 @@ logfile_maxbytes=1MB
 logfile_backups=2
 nodaemon=true
 
+[eventlistener:supervisor-proc-exit-listener]
+command=/usr/bin/supervisor-proc-exit-listener
+events=PROCESS_STATE_EXITED
+autostart=true
+autorestart=unexpected
+
```

Comment:
make rsyslogd to restart automatically.

Reply:
Done in https://github.com/Azure/sonic-buildimage/pull/2546/commits/09fd522ef8969f09a2a5ee5e62c5f770b0ba5387